test/s3path.jl

```diff
@@ -0,0 +1,202 @@
+root = Path("s3://$bucket_name/pathset-root/")
+ps = PathSet(
+    root,
+    root / "foo/",
```

Comment:
Just curious if `root / "foo" /` works?

Reply:
Thinking about this more I realized we `/` to be supported as a post-fix operator.